### ISSUE TITLE
docker sbt build layer is huge due to build dependencies left in FS

### ISSUE THREAD

**Issue by foxel** [bwsw/sj-platform#14]
In REST docker image the layer with `sbt sj-crud-rest/assembly` is huge due to build dependencies:

```
$ docker history bwsw/sj-rest
...
<missing>           2 days ago          /bin/sh -c sbt sj-crud-rest/assembly            696 MB 
...
```

This can be reduced by removing sbt build dependencies **in same RUN call**

**Comment by Wombasta** [comment]
Fixed in 1.0.2

**Comment by Wombasta** [comment]
695.8 MB ->170.1 MB
